web/compile_wasm.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail

```

Just some bash defensive programming: https://buildkite.com/docs/pipelines/writing-build-scripts#configuring-bash

Reply:
pretty sweet!